gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.2-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.0-all.zip
```

Comment:
This should stay as bin, gradle-6.0-bin.zip.

Reply:
Looking at it but struggling with compiler errors in the examples. Are they for a future 1.30.0 version?